app/Http/Controllers/UserController.php

```diff
@@ -32,16 +35,47 @@ public function create(Request $request)
     public function store(UserRequest $request)
     {   
         $user_token = $request->authentication_token;
+
         $authentication = Authentication::where('token', $user_token)
             ->where('status', AuthenticationStatus::MAIL_SENT)
             ->where('expired_at', '>', Carbon::now())
             ->first();
+
         if(is_null($authentication)){
             return to_route('tasks.index')->withErrors(['status_error' => '会員登録に失敗しました。']);
         }
+
+        $image = new Imagick();
+        $image->readImage($request->file('image_file'));
         
+        $archive_image_path = 'archive_images/'.Str::random(rand(20,50)).'.webp';
+        while(!is_null(User::where('archive_image_path',$archive_image_path)->first())){
```

Comment:
```suggestion
        while(!is_null(User::where('archive_image_path', $archive_image_path)->first())){
```
メソッド第二引数のカンマ後は半角スペース開けるようにしてください。

Reply:
半角スペースを開ける修正を行いました。
ご確認宜しくお願い致します。
app/Http/Controllers/UserController.php